lib/core/client.js

```diff
@@ -700,10 +702,31 @@ function connect (client) {
   if (protocol === 'https:') {
     servername = client[kServerName] || (client[kTLSOpts] && client[kTLSOpts].servername)
     const tlsOpts = { ...client[kTLSOpts], servername }
+    const sessionName = createName(client[kUrl], tlsOpts)
+    // Get cached session if it does exists
+    const session = client[kTLSSessions].get(sessionName)
+    if (session !== undefined) tlsOpts.session = session
```

Comment:
style

Reply:
move down `tlsOpts =` and initialize it directly?

Reply:
What do you mean by "style"? We are using [standard](https://standardjs.com/) and the code above is valid.

Reply:
> move down tlsOpts = and initialize it directly? const tlsOpts = { ...client[kTLSOpts], servername, session }

I can't, as `createName` needs tls options.

Reply:
> What do you mean by "style"? We are using standard and the code above is valid.

Use braces please.